test/assignment-tests.js

```diff
@@ -0,0 +1,80 @@
+import test from 'tape';
+import compose from '../examples/compose';
+
+const assignmentProps = [
+  'methods',
+  'properties',
+  'staticProperties',
+  'deepStaticProperties',
+  'propertyDescriptors',
+  'staticPropertyDescriptors'
+];
+
+const build = (num) => {
+  const composable = {};
+
+  assignmentProps.forEach(prop => {
+    composable[prop] = {
+      [num]: num,
+      override: num
+    };
+  });
+
+  return { compose: composable };
```

Comment:
`compose()` can take stamps or POJO descriptors, but you're partially right. Here I should either be sending in functions, or top level POJO descriptors, though I think it would be a mistake to assume you're getting a function.

> Composable descriptor (or just Descriptor) is a meta data object which contains the information necessary to create an object instance. Descriptor properties are attached to the .compose() method, e.g. .compose.methods